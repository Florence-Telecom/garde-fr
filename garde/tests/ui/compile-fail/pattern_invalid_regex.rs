#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(pattern("("))]
    field: &'a str,
}

fn main() {}
