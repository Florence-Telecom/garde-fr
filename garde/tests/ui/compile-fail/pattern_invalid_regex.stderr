error: invalid regex: regex parse error:
           (
           ^
       error: unclosed group
 --> tests/ui/compile-fail/pattern_invalid_regex.rs:3:21
  |
3 |     #[garde(pattern("("))]
  |                     ^^^
